release-please-config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+  "packages": {
+    ".": {
+      "changelog-path": "CHANGELOG.md",
+      "changelog-type": "github",
+      "release-type": "node",
+      "bump-minor-pre-major": false,
+      "bump-patch-for-minor-pre-major": false,
+      "draft": false,
+      "prerelease": false,
+      "extra-files": [
+        { "type": "json", "path": "jsr.json", "jsonpath": "$.version" }
+      ]
+    }
+  },
+  "changelog-sections": [
```

Comment:
do we need this "changelog-sections" defined? can we just use the default?

Reply:
The default values were not well documented, so I thought to set those explicitly. 

Found it now in their code. 
https://github.com/googleapis/release-please/blob/422df5a6a9d44a52c6c682563d1795b8fd246215/docs/manifest-releaser.md?plain=1#L185